examples/wrapper-components/react-javascript/src/App.js

```diff
@@ -1,31 +1,26 @@
 import './App.css';
-import Alert from './components/Alert/Alert'
+import Alert from './components/Alert.vue'
```

Comment:
Again, please make sure to use the correct import paths and test before you commit your changes